testing_application/app.py

```diff
@@ -0,0 +1,89 @@
+from typing import TypedDict
+
+from robyn import Robyn, Request, jsonify, OpenAPI, SubRouter
+from robyn.openapi import OpenAPIInfo, Contact, License, ExternalDocumentation, Components
+
+# this is a file currently for testing the implementation -- will be removed once the PR is ready for merging
+
+pet_sample_app = Robyn(
+    file_object=__file__,
+    openapi=OpenAPI(
+        info=OpenAPIInfo(
+            title="Sample Pet Store App",
+            description=" This is a sample server for a pet store.",
+            termsOfService=" https://example.com/terms/",
+            version=" 1.0.1",
+            contact=Contact(
+                name="API Support",
+                url="https://www.example.com/support",
+                email="support@example.com",
+            ),
+            license=License(
+                name="Apache 2.0",
+                url="https://www.apache.org/licenses/LICENSE-2.0.html",
+            ),
+            servers=[
+                # Server(
+                #     url="//",
+                #     description="Debug environment"
+                # ),
+                # Server(
+                #     url="https://example.com/api/v1/",
+                #     description="Production environment"
+                # ),
```

Comment:
Do we need this?

Reply:
need? no, but I just demonstrated an example the user can use. anyways, that file is to be removed later, as specified in the comment. I will remove it for now.